addons/web/static/src/webclient/commands/command_palette.xml

```diff
@@ -52,7 +44,7 @@
   <t t-name="web.hotkeyCommandItem" owl="1">
```

Comment:
wrong template naming convention ?

I think it should be:
`module_name_in_snake_case.TemplatePartInPascalCase`